client/app/index.html

```diff
@@ -9,6 +9,7 @@
     <link rel="icon" type="image/png" sizes="32x32" href="./assets/images/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="96x96" href="./assets/images/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="./assets/images/favicon-16x16.png">
+    <link rel="icon" href="./assets/images/favicon.ico">
```

Comment:
The problem was wrong paths, but once they are sorted I don't think we need reference for "regular" favicon. See 3c7c93fc9f09cfa17319aef14da3d23ec6e46641.